crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -176,4 +176,17 @@ public boolean isCds15WorkAround(final RepOrderDTO repOrderDTO) {
         return PassportAssessmentResult.FAIL.getResult().equals(passportAssessmentResult)
                 && InitAssessmentResult.PASS.getResult().equals(initialAssessmentResult);
     }
+
+    public boolean hasMessageOutcomeChanged(int repId, String laaTransactionId, String msgOutcome) {
+
+        RepOrderDTO repOrderDTO = maatCourtDataService.getRepOrderByRepId(repId, laaTransactionId);
+        if (null != repOrderDTO) {
+
+            String messageOutcome = Optional.ofNullable(repOrderDTO.getMagsOutcome()).orElse("na");
+            if (!messageOutcome.equals(msgOutcome)) {
```

Comment:
This can be simplified as return !messageOutcome.equals(msgOutcome);

Reply:
Thanks. Done